test/ApplicationSpec.scala

```diff
@@ -9,11 +9,6 @@ import play.api.test.Helpers._
 class ApplicationSpec extends Specification {
 
   "Application" should {
-
-    "send 404 on a bad request" in new WithApplication {
```

Comment:
The '404 on a bad request' test (a sample test supplied by the old version of Play) doesn't work anymore - slightly different behaviour of the Playframework - so have just deleted it.